cv32e40s/regress/cv32e40s_full.yaml

```diff
@@ -406,11 +406,10 @@ tests:
     dir: cv32e40s/sim/uvmt
     cmd: make test TEST=b_ext_test
 
-#TODO:MT Comment back in when Zc is enabled in the core
-#  zc_test:
-#    description: Zc directed test
-#    builds: [ uvmt_cv32e40s, uvmt_cv32e40s_pma_1]
-#    dir: cv32e40s/sim/uvmt
-#    cmd: make test TEST=illegal
-#    num: 5
-#
+  zc_test:
+    description: Zc directed test
+    builds: [ uvmt_cv32e40s, uvmt_cv32e40s_pma_1]
+    dir: cv32e40s/sim/uvmt
+    cmd: make test TEST=zc_test
+    num: 5
```

Comment:
What does "5" mean here? Is it run 5 times for "uvmt_cv32e40s" and 5 times for "uvmt_cv32e40s_pma_1"? But it is a directed test, so what is the purpose of running it 5 times for each config?

Reply:
My two cents:
1. Yes, "num" is the number of runs of this test.  (Maybe "runs" would have been better choice.)
2. Even if the test-program is static, the simulation _could_ be made random by enabling variable cycle delays and/or errors on the OBI, or even by enabling interrupt/debug events.

Reply:
> But it is a directed test, so what is the purpose of running it 5 times for each config?

> Even if the test-program is static, the simulation could be made random by enabling variable cycle delays and/or errors on the OBI, or even by enabling interrupt/debug events.

@silabs-mateilga could you comment on this?

--------

Quick link for convenience:
https://github.com/openhwgroup/core-v-verif/tree/cv32e40s/dev/cv32e40s/tests/programs/custom/zc_test

Reply:
Is it because of the `vp_random_num` and `glb_irq_delay` etc?

Reply:
The test has randomness built into it, so there is definitely a value in running it more than once. The timing of the interrupt hitting each multioperation instruction is randomized.